translations/strings.yaml

```diff
@@ -366,6 +366,9 @@ connectionInfo:
   labelPing:
     value: Ping
     comment: Latency of the connection
+  labelUpload:
+    value: Upload
```

Comment:
How/where it this used? It would be helpful to add that information to the comment